engine/src/main/scala/cromwell/jobstore/JobStoreReaderActor.scala

```diff
@@ -20,15 +21,24 @@ class JobStoreReaderActor(database: JobStore, override val serviceRegistryActor:
     with ActorLogging {
 
   override def processHead(head: CommandAndReplyTo[QueryJobCompletion]) = instrumentedProcess {
+    log.info("{}: Processing Job store check", head.command.jobKey.tag)
+
     val action = database.readJobResult(head.command.jobKey, head.command.taskOutputs)
+
     action onComplete {
-      case Success(Some(result)) => head.replyTo ! JobComplete(result)
-      case Success(None) => head.replyTo ! JobNotComplete
+      case Success(Some(result)) =>
+        head.replyTo ! JobComplete(result)
+      case Success(None) =>
+        head.replyTo ! JobNotComplete
       case Failure(t) =>
         log.error(t, "JobStoreReadFailure")
         head.replyTo ! JobStoreReadFailure(t)
     }
-    action.map(_ => 1)
+
+    for {
+      actionResult <- action
+      _ = log.info(s"{}: Job store check success (found something = ${actionResult.nonEmpty})", head.command.jobKey.tag)
```

Comment:
... or you could ignore my earlier comment and put this log message in the `Success` cases...

Reply:
That was my original layout. But I think this way is possibly nicer?

Reply:
yeah this is fine